quantization-example.sh

```diff
@@ -29,7 +29,7 @@ echo "Training..."
 ./fasttext supervised -input "${DATADIR}/dbpedia.train" -output "${RESULTDIR}/dbpedia" -dim 10 -lr 0.1 -wordNgrams 2 -minCount 1 -bucket 10000000 -epoch 5 -thread 4
 
 echo "Quantizing..."
-./fasttext quantize -output "${RESULTDIR}/dbpedia" -input "${DATADIR}/dbpedia.train" -qnorm -retrain -epoch 1 -cutoff 100000
+./fasttext quantize -input "${RESULTDIR}/dbpedia" -input "${DATADIR}/dbpedia.train" -qnorm -retrain -epoch 1 -cutoff 100000
```

Comment:
Should it be just the input for model but not training data?

`./fasttext quantize -input "${RESULTDIR}/dbpedia" -qnorm -retrain -epoch 1 -cutoff 100000`

Reply:
Yes I think that's absolutely right!